src/Compilers/VisualBasic/Portable/Scanner/CharacterInfo.vb

```diff
@@ -257,7 +257,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Friend Shared Function BeginsBaseLiteral(c As Char) As Boolean
             Return (c = "H"c Or c = "O"c Or c = "B"c Or c = "h"c Or c = "o"c Or c = "b"c) OrElse
-                    (IsFullWidth(c) AndAlso (c = FULLWIDTH_LATIN_CAPITAL_LETTER_H Or c = FULLWIDTH_LATIN_CAPITAL_LETTER_O Or c = FULLWIDTH_LATIN_CAPITAL_LETTER_B Or c = FULLWIDTH_LATIN_SMALL_LETTER_H Or c = FULLWIDTH_LATIN_SMALL_LETTER_O Or c = FULLWIDTH_LATIN_CAPITAL_LETTER_B))
+                    (IsFullWidth(c) AndAlso FullwidthCharacter_BeginsBaseLiteral(c))
+        End Function
+
+        Private Shared Function FullwidthCharacter_BeginsBaseLiteral(c As Char) As Boolean
```

Comment:
>FullwidthCharacter_BeginsBaseLiteral [](start = 32, length = 36)

Please get rid of this function, just fix the typo in place.

Reply:
Why? The refactoring is smaller in size for the common case. *(which increase the chances of JIT Inlining.)* The less common usage case of fullwidth characters being in separate function make it a candidate for inlining also. Combining them into one, eliminates it for both cases, so is worse off.

Reply:
I am not aware of any problem that the original function had with inlining. Was there a proven problem? Is there a proof that things got better?

Reply:
Would inlining even make any difference?